functest/logicrunner_test.go

```diff
@@ -381,9 +385,19 @@ func (r *Two) Value() (int, error) {
 	resp := callMethod(t, obj, "Hello")
 	require.Empty(t, resp.Error)
 
-	resp = callMethod(t, obj, "Value")
-	require.Empty(t, resp.Error)
-	require.Equal(t, float64(644), resp.ExtractedReply)
+	testPassed := false
+	for i := 0; i < 25; i++ {
+		resp = callMethod(t, obj, "Value")
+		require.Empty(t, resp.Error)
+		fmt.Println(">>>> resp.ExtractedReply: ", resp.ExtractedReply)
```

Comment:
are you sure you want to `Println` here?

Reply:
fixed